docs/ref/models/querysets.txt

```diff
@@ -1146,6 +1139,18 @@ Chaining ``select_related`` calls works in a similar way to other methods -
 that is that ``select_related('foo', 'bar')`` is equivalent to
 ``select_related('foo').select_related('bar')``.
 
+There may be some situations where you wish to call ``select_related()`` with a
+lot of related objects, or where you don't know all of the relations. In these
+cases it is possible to call ``select_related()`` with no arguments. This will
+follow all non-null foreign keys it can find - nullable foreign keys must be
```

Comment:
Wow, I had no idea this went beyond a depth of 1. 🤯  I see more value now in removing this.